test/passes/Oz_fuzz-exec_all-features.wast

```diff
@@ -325,6 +325,42 @@
    )
   )
  )
+ (func "ref-as-data-of-func"
+  (drop
+   ;; This should trap.
+   (ref.as_data
+    (ref.func $0)
+   )
+  )
+ )
+ (func "ref-as-data-of-data"
+  (drop
+   ;; This should trap.
```

Comment:
Why?

Reply:
Oh, that looks like a bad copy-paste, sorry...